client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -129,6 +141,10 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
 			return (
 				<div>
 					<QuerySiteSettings siteId={ this.props.siteId } />
+					{
+						this.props.isJetpack &&
+						<QueryJetpackSettings siteId={ this.props.siteId } />
```

Comment:
Ah, good catch. Yeah, this had to be everywhere where we perform Jetpack actions (both requesting and updating). Addressed in https://github.com/Automattic/wp-calypso/pull/10697/commits/0ccb42c36f6315a5a1ab2a358515fb52eeea930d.